src/schemathesis/hooks.py

```diff
@@ -222,6 +222,11 @@ def before_load_schema(context: HookContext, raw_schema: Dict[str, Any]) -> None
     """Called before schema instance is created."""
 
 
+@HookDispatcher.register_spec([HookScope.GLOBAL])
+def before_one(strategy: st.SearchStrategy) -> st.SearchStrategy:
```

Comment:
Added `context` parameter. I made `context` the second parameter to match the format of `before_generate*(strategy: st.SearchStrategy, context: HookContext)`.

Reply:
Oh ... I completely forgot to update the docs. In the actual code, it is passed as the first argument :( I will update the docs

Reply:
That's all right! I will update the signatures in this PR (if we want the fix merged into master faster in another PR, I'll just consider it an effort to avoid merge conflicts).

Reply:
Cool :) Fixed in master in e846790fdf988d9ac836c948d9d6599329077c72